serverv2/cometbft/types/abci.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	"context"
+
+	abci "github.com/cometbft/cometbft/abci/types"
+)
+
+// ABCI is an interface that enables any finite, deterministic state machine
+// to be driven by a blockchain-based replication engine via the ABCI.
+type ABCI interface {
```

Comment:
i dont think this is needed, or not sure what its used for

Reply:
stopped using it. Removed 👌